collections/_services/certbot.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: service
+type: service
+
+title: Certbot
+version: ''
+order: 6
+description: >-
+   Use Certbot to automatically update TLS Certificates on OpenShift Routes.
+descriptionLinks:
+  - link:
+      url: https://github.com/BCDevOps/certbot
+      text: Github
+  - link:
+      url: https://github.com/BCDevOps/certbot/blob/master/README.md
+      text: Review Certbot documentation
+contentStyle: unstructured
+onboard:
+  - 'Self Hosted Service'
+  - 'DockerHub'
+onboardDescription: We provide a <em>Docker Image</em> as well as documented <em>open-source code</em> for hosting your own service.
+urls:
+  github: https://github.com/BCDevOps/certbot
+  docs: https://github.com/BCDevOps/certbot/blob/master/README.md
+dockerImages:
+  - name: Certbot
+    url: https://hub.docker.com/r/bcgovimages/certbot/
+    description: This image provides ???
+    badges:
+      - https://img.shields.io/docker/v/bcgovimages/certbot.svg?sort=semver
+      - https://img.shields.io/docker/pulls/bcgovimages/certbot.svg
+      - https://img.shields.io/docker/image-size/bcgovimages/certbot.svg
+---
+
+#### Feature List
+
+- Can utilize [https://letsencrypt.org/](https://letsencrypt.org/){:target="_blank"} or other ACME compliant Certificate Authority for issuing certificates
+- Leverages and extends [https://certbot.eff.org/](https://certbot.eff.org/){:target="_blank"} for managing (create/renew) certificates
+- Should only be executed on Openshift Container Platform
+- Creates an OpenShift CronJob which will run on a regular schedule for renewing TLS certificates
+  - The CronJob will manage all Route objects annotated with the label certbot-managed=true
+  - One certificate will be issued/renewed for all the managed hosts/domains
+- If a cert is created/renewed, patch the new certificate to the managed OpenShift routes
+
+#### Self-hosted
+
+If you want to host your own version of Certbot, you can start by:
```

Comment:
because they can ONLY host their own version maybe this could say something more like:
'Read the documentation on the GitHub README(link) to learn what you need to do to deploy Certbot'
bit redundant i guess

Reply:
Changed  'host' to 'deploy'